VideoLocker/default_config/config.yaml

```diff
@@ -1,4 +1,4 @@
-API_HOST_URL: 'http://localhost:8000'
+API_HOST_URL: 'http://localhost:8000/mock/'
```

Comment:
This shouldn't be changed. We want it to point to a live localhost setting. If we need this for testing, we should have the tests set up a valid config.